yt_dlp/extractor/adn.py

```diff
@@ -147,6 +158,8 @@ def _perform_login(self, username, password):
             self.report_warning(message or self._LOGIN_ERR_MESSAGE)
 
     def _real_extract(self, url):
+        if 'network.de' in url:
+            self._HEADERS['X-Target-Distribution'] = 'de'
```

Comment:
Don't add this to _HEADERS. The same IE class is used for multiple URLs. Just add the header locally to relevant calls